app/routes/index.md

```diff
@@ -3,4 +3,47 @@ meta:
   title: GCN - General Coordinates Network
 ---
 
-# Welcome to the General Coordinates Network
+# Welcome to the General Coordinates Network (GCN)
+
+<img src="https://gcn.gsfc.nasa.gov/GCN.gif" width="400" align="right"/>
+
+GCN is the NASA broker for transient alerts and community reports from space- and ground-based missions, observatories, and experiments used by the multimessenger and time-domain astrophysics communities. GCN is the primary distributer of detections and follow-up observations of Gamma-ray Bursts (GRBs), gravitational wave compact binary mergers, and high-energy neutrinos.
```

Comment:
"Broker" is a a pretty jargon-y term, and it's only in common use by people who are already deeply embedded in the Kafka and astronomical transients community. Can we explain what we do without using the term "broker", and instead use terms like "community" and "collaboration platform"?